tools/uw-frame-static/build.js

```diff
@@ -12,9 +12,12 @@ var exec_handler = function(error, stdout, stderr) {
 
 var mkdirp = require('mkdirp');
 var copy = require('recursive-copy');
+var copyOptions = {
+  overwrite: true
+}
 
-mkdirp('uw-frame-static/target/css/themes/', function (err) {
-  if (err) throw error;
+mkdirp('uw-frame-static/target/css/themes/', function (error) {
+  if (error) throw error;
```

Comment:
I want to trust that `Error` would be immediately noticeable due to classes starting with capitals by convention.  If someone saw `Error` and not `Error()` or `new Error()` that should be picked up during review.